doc/code/qml.rst

```diff
@@ -6,4 +6,4 @@ qml
 .. automodapi:: pennylane
     :no-heading:
     :include-all-objects:
-    :skip: iter_entry_points, Version, SimpleSpec, plugin_devices, plugin_converters, default_config, reload
+    :skip: iter_entry_points, Version, SimpleSpec, plugin_devices, plugin_converters, default_config, reload, X, P
```

Comment:
Does this remove it completely from the docs?

Reply:
It removes it from the qml page, but it's possible I missed something somewhere. I also had to remove it separately from the Operators intro page. If it shows up somewhere other than those two pages, its still there. But I didn't find it anywhere else. It is admittedly harder to search for than most things 😆

Reply:
Thanks! To clarify, I don't think we should remove it from the docs at this stage - users should be able to see `X` and `P` in the docs and read the warning about its name changing. Indeed, that's what we already have [here](https://github.com/PennyLaneAI/pennylane/pull/4330/files#diff-beac1f4626960fd49cbd9c6fef0956bc07957e80117c1081221e20e4ce0c0c71R1170), we just need to show it.

Reply:
Sounds good, I'll put it back!